src/http-gateways/trustless-gateway.md

```diff
@@ -257,6 +290,15 @@ Same as in :cite[path-gateway], SHOULD be returned when Trustless Gateway
 supports more than a single response format and the `format` query parameter is
 missing or does not match well-known format from `Accept` header.
 
+### `Retry-After` (response header)
```

Comment:
Update https://specs.ipfs.tech/http-gateways/path-gateway/#retry-after-response-header and refer to that.